packages/account/src/Types/onfido-phrases.type.ts

```diff
@@ -0,0 +1,313 @@
+        title_id_front: JSX.Element;
+        title_license_back: JSX.Element;
+        title_license_front: JSX.Element;
+        title_passport: JSX.Element;
+        title_permit_back: JSX.Element;
+        title_permit_front: JSX.Element;
+    };
+    error_unsupported_browser: {
+        subtitle_android: JSX.Element;
+        subtitle_ios: JSX.Element;
+        title_android: JSX.Element;
+        title_ios: JSX.Element;
+    };
+    generic: {
+        accessibility: {
+            close_sdk_screen: JSX.Element;
+            dismiss_alert: JSX.Element;
+        };
+        back: JSX.Element;
+        close: JSX.Element;
+        errors: {
+            interrupted_flow_error: {
+                instruction: JSX.Element;
+                message: JSX.Element;
+            };
+            invalid_size: {
+                instruction: JSX.Element;
+                message: JSX.Element;
+            };
+            invalid_type: {
+                instruction: JSX.Element;
+                message: JSX.Element;
+            };
+            lazy_loading: {
+                message: JSX.Element;
+            };
+            multiple_faces: {
+                instruction: JSX.Element;
+                message: JSX.Element;
+            };
+            no_face: {
+                instruction: JSX.Element;
+                message: JSX.Element;
+            };
+            request_error: {
+                instruction: JSX.Element;
+                message: JSX.Element;
+            };
+            sms_failed: {
+                instruction: JSX.Element;
+                message: JSX.Element;
+            };
+            sms_overuse: {
+                instruction: JSX.Element;
+                message: JSX.Element;
+            };
+            unsupported_file: {
+                instruction: JSX.Element;
+                message: JSX.Element;
+            };
+        };
+        lazy_load_placeholder: JSX.Element;
+        loading: JSX.Element;
+    };
+    get_link: {
+        alert_wrong_number: JSX.Element;
+        button_copied: JSX.Element;
+        button_copy: JSX.Element;
+        button_submit: JSX.Element;
+        info_qr_how: JSX.Element;
+        info_qr_how_list_item_camera: JSX.Element;
+        info_qr_how_list_item_download: JSX.Element;
+        link_divider: JSX.Element;
+        link_qr: JSX.Element;
+        link_sms: JSX.Element;
+        link_url: JSX.Element;
+        loader_sending: JSX.Element;
+        number_field_input_placeholder: JSX.Element;
+        number_field_label: JSX.Element;
+        subtitle_qr: JSX.Element;
+        subtitle_sms: JSX.Element;
+        subtitle_url: JSX.Element;
+        title: JSX.Element;
+        url_field_label: JSX.Element;
+    };
+    linked_computer: {
+        button_primary: JSX.Element;
+        info: JSX.Element;
+        list_item_desktop_open: JSX.Element;
+        list_item_sent_by_you: JSX.Element;
+        subtitle: JSX.Element;
+        title: JSX.Element;
```

Comment:
is there any reason for writing JSX.Element for all? could this be a string?

Reply:
The file uses <localize> component which returns a JSX Element